detekt-rules-style/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/style/VarCouldBeValSpec.kt

```diff
@@ -13,6 +13,82 @@ class VarCouldBeValSpec : Spek({
     val env: KotlinCoreEnvironment by memoized()
     val subject by memoized { VarCouldBeVal() }
 
+    describe("file-level declarations") {
+        it("does not report non-private variables") {
+            val code = """
+                var a = 1
+            """.trimIndent()
+
+            assertThat(subject.compileAndLintWithContext(env, code)).isEmpty()
+        }
+
+        it("reports private variables that are never re-assigned") {
+            val code = """
+                private var a = 1
+                
+                fun foo() {
+                    val fizz = "BUZZ"
+                }
+            """.trimIndent()
+
+            assertThat(subject.compileAndLintWithContext(env, code)).hasSize(1)
+        }
+
+        it("does not report private variables that are re-assigned") {
+            val code = """
+                private var a = 1
+
+                fun foo() {
+                    a = 2
+                }
+            """
+
+            assertThat(subject.compileAndLintWithContext(env, code)).isEmpty()
+        }
+    }
+
+    describe("class-level declarations") {
+        it("does not report non-private variables in non-private classes") {
```

Comment:
I guess the following is a bit easier to read and grasp.

```suggestion
        it("does not report public variables in public classes") {
```